NEWS.md

```diff
@@ -81,6 +81,10 @@ core developer team.
 
 * ggplot2 now works in Turkish locale (@yutannihilation, #3011).
 
+* Clearer error messages for inappropriate aesthetics (@clairemcwhite, #3060).
+
+* `geom_rug()` gains an "outside" option to allow for moving the rug tassels to outside the plot area. (@njtierney, #3085)
```

Comment:
Do we have an idea of where this news item comes from? It should have been present in NEWS.md already, but it isn't, so I'm confused about both why it was lost and how it made its reappearance here.

Reply:
Ah, it was moved in this commit: https://github.com/tidyverse/ggplot2/commit/230e8f78ae54d255aa02ad2f2d7efca9c46905d3 so you can just delete it here.

Reply:
Got it, will do in next commit.